modules/azure/mssql/outputs.tf

```
@@ -30,3 +30,7 @@ output "admin_password" {
 output "principal_id" {
   value = azurerm_mssql_server.mssql_server.identity[0].principal_id
 }
+
+output "private_ip_address" {
+  value = data.azurerm_private_endpoint_connection.private_endpoint_connection.private_service_connection.0.private_ip_address
```

Can the same not be done by doing this?

```suggestion
  value = data.azurerm_private_endpoint.private_endpoint.private_service_connection.0.private_ip_address
```

Than we don't need to add a extra `data "azurerm_private_endpoint_connection"`.

https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/private_endpoint#private_ip_address

---

Might be, but at least we know this works

---

@jeroenvleeming @bartwesselink Well we can just test it?

I would say my solution is more efficient, with less resource/data calls, what results in less github action minutes used, so is cheaper.

Just add a extra output that uses `data.azurerm_private_endpoint.private_endpoint.private_service_connection.0.private_ip_address` and compare the ip's, if they are the same we put in my change.